configs/config_ref_example.json

```diff
@@ -0,0 +1,96 @@
+{
+  "servers": [
+    {
+      "default_log_path": "stdout",
+      "default_log_level": "DEBUG",
+      "routers": [
+        {
+          "router_config_name": "actuator_dont_need_router"
+        }
+      ],
+      "listeners": [
+        {
+          "name": "grpc",
+          "address": "127.0.0.1:34904",
+          "bind_port": true,
+          "filter_chains": [
+            {
+              "filters": [
+                {
+                  "type": "grpc",
+                  "config": {
+                    "server_name": "runtime",
+                    "grpc_config": {
+                      "hellos": {
+                        "helloworld": {
+                          "type": "helloworld",
+                          "hello": "greeting"
+                        }
+                      },
+                      "secret_store": {
+                        "local.file": {
+                          "type":"local.file",
+                          "metadata": {
+                            "secretsFile": "../../configs/secret/config_secret_local_file.json"
+                          }
+                        },
+                        "local.env": {
+                          "type":"local.env",
+                          "metadata": {
+                          }
+                        }
+                      },
+                      "sequencer": {
+                        "redis": {
+                          "type":"redis",
+                          "metadata": {
+                            "redisHost": "127.0.0.1:6380",
+                            "redisPassword": ""
+                          },
+                          "secret_ref": [
+                            {
+                              "name": "local.file",
+                              "key": "redisPassword"
```

Comment:
Wait ! I just thought of a question
This `key` here represent the key in the secret store.... but how do we specify the key injected in the metadata ?